Feat/button component

frankkim92 opened the pull request:
## What is this PR? :mag:
CustomButton을 사용할 수 있습니다.
CustomInput을 사용할 수 있습니다.
-

## branch

- feature/button-component -> develop

## Changes :memo:

- 커스텀 버튼 컴포넌트가 추가되었습니다.
- 커스텀 인풋 컴포넌트가 추가되었습니다.
- ButtonComponents에서 CustomButton으로 파일명이 변경되었습니다.
- ButtonComponents에서 disabled 시 Cursor와 Opacity 효과를 주었습니다.

by @frankkim92 #1 #2 



Reviewers
@Jeremy-Kr 
@harenohee 
@dollorofkorea 
@ambition0103

frankkim92 committed:
feat :  Update Reusable ButtonComponent
재사용 가능한 CustomButton을 생성하였습니다.
@by frankkim92 #1

frankkim92 committed:
feat : Update Hover Reusable ButtonComponent
 CustomButton에 Hover 기능을 추가하였습니다.
@by frankkim92 #1

frankkim92 committed:
feat : Update Border-Size

 CustomButton의 Border-Size px를 조정했습니다.
@by frankkim92 #1

Jeremy-Kr commented on src/components/common/ButtonComponents.tsx:
```diff
@@ -0,0 +1,57 @@
+
+  background: ${(props) =>
+    props.backgroundColor
+      ? (props: any) => props.theme.colors[`${props.backgroundColor}`]
```
요부분 제가 한번 고쳐볼게요

Jeremy-Kr replied:
못했읍니다..

frankkim92 replied:
앗.. 감사합니다.

frankkim92 replied:
정익님, 말씀해주신대로 파일명 CustomButton으로 수정을 완료했습니다.
또한 CustomInput 기능도 커밋 완료하였습니다.

Jeremy-Kr committed:
fix: interface ButtonProps

StyledComponents의 props에서 theme은 항상 들어오는 값이므로 ?를 삭제하였습니다.

by @Jeremy-Kr  #1

Jeremy-Kr reviewed (approved):
LGTM

harenohee reviewed (dismissed):
귯 확인했습니당

KoreaMoney reviewed (dismissed):
고생많으셨습니다!

Jeremy-Kr reviewed (changes requested):
엄... 지금 보니 파일명이 ButtonComponents로 되어있는데 CustomButton이 더 적절 해 보입니다! 변경해서 커밋 부탁드려요

frankkim92 committed:
feat : Update Reusable InputComponent
- 재사용 가능한 CustomInput을 생성하였습니다.
- ButtonComponent 파일명을 CustomButton으로 변경하였습니다.
@by frankkim92 #2

frankkim92 committed:
Merge branch 'feat/button-component' of https://github.com/money-fragments/money-fragments-client into feat/button-component

frankkim92 commented:
> 엄... 지금 보니 파일명이 ButtonComponents로 되어있는데 CustomButton이 더 적절 해 보입니다! 변경해서 커밋 부탁드려요

변경 완료하였습니다.

Jeremy-Kr reviewed (dismissed):
LGTM 커스텀 인풋도 같이 들어가있네요! 다른 팀원분들도 확인 해 주시고, 혹시 placeholder는 따로 지정 안해줘도 될까요? 확인 부탁드려요! @frankkim92

harenohee reviewed (dismissed):
커스텀인풋 확인했습니다. 🥹

frankkim92 committed:
feat : Update Reusable ButtonComponent

- 재사용 가능한 버튼 컴포넌트 내에서, Disabled시 Cursor는 not-allowed를 주었고 Opacity는 0.5를 주었습니다.

@by frankkim92 #2

ambition0103 commented:
고생하셨습니다 .. 굿

Jeremy-Kr reviewed (approved):
LGTM

ambition0103 reviewed (approved):
고생하셨습니다.
